src/assets/20-tech-terms/20-beginner-tech-terms.md

```diff
@@ -0,0 +1,157 @@
+Key Features:
+- `Scalability`: Easy to extend and adapt for interoperation with other systems and applications
+- `Efficiency`: Can be embedded in other applications with great ease, thus giving you the freedom to utilize present functionality again instead of writing code from scratch.
+- `Security`: Ensures secure ways of accessing resources and a healthy trade of data.
+
+### 18. Cookies and Sessions
+`Cookies` and `sessions` are tools to store user data/state in our web applications. It enables websites to remember their own users so as to create a more personalized experience.
+
+**Cookies**:
+- **Meaning**: It is the tiny piece of data that is stored by a website on your browser.
+- **Goal**: For the purpose of remembering user activity, login details and preferences.
+- **Example**: Logging-in to certain websites, and checking the "remember me" box will store a cookie in your web browser.
+
+**Sessions**:
+- **Meaning**: Some type of temporary storage which includes server-side and client-side components that are used to store user data during the visit from the time a visitor sits on page until when they leave (session).
+- **Goal**: To maintain the state of the user and data during navigation across a website.
+- **Example**: When you add items to a crt on an e-commerce site, a session is created to keep the item in the cart until payment is finalized or the user leaves the site.
+
+## Authentication:
+
+### 19. SSO:
+`Single Sign-On,SSO` in short is an authentication process which allows users to enter one set of login credentials for multiple applications. It simplifies the user experience by removing the requirement to remember unique usernames and passwords for each application. That is nice for users but it also improves the security of your application and makes controlling user access easier by having one central location where validation occurs.
+Example Scenario:
+Suppose, you are logging into your company's intranet. `SSO` allows you to log into email, project management accounts and some other tools without having to login for each.
+`SSO` simplifies the log-in process and boosts user efficiency while increasing security as it centralizes credential management.
+
+## Documentation:
+
+### 20. User Documentation:
+`User documentation` is a high level guide for assisting users to understand and utilize products or services.  It provides examples, FAQs, explanations, and troubleshooting advice making the service easily navigable and user-friendly.
+
+Picture yourself learning how to use a new code library for the first time as a developer and it would be overwhelming without clear instructions. This is where User documentation saves the day! It includes step-by-step guides for how to integrate the library, answers the most common questions and provides solutions for typical problems.
```

Comment:
```suggestion
Picture yourself learning how to use a new code library for the first time as a developer and it would be overwhelming without clear instructions. This is where user documentation saves the day! It includes step-by-step guides for how to integrate the library, answers the most common questions and provides solutions for typical problems.
```
or 

```suggestion
Picture yourself learning how to use a new code library for the first time as a developer and it would be overwhelming without clear instructions. This is where `User documentation` saves the day! It includes step-by-step guides for how to integrate the library, answers the most common questions and provides solutions for typical problems.
```